app/src/main/res/values-de/strings.xml

```diff
@@ -0,0 +1,102 @@
+<resources>
+    <string name="app_name">Private Location</string>
+    <string name="app_desc">Open-Source Location Privacy</string>
+
+    <string name="app_github_dev">https://github.com/wesaphzt</string>
+    <string name="app_github">https://github.com/wesaphzt/PrivateLocation</string>
+
+    <!--actionbar-->
+    <string name="action_favorites">Favoriten</string>
+    <string name="action_go_to">Gehe zu</string>
+    <string name="action_settings">Einstellungen</string>
+    <string name="action_donate">Spenden</string>
+    <string name="action_show_intro">Einführung</string>
+    <string name="action_about">Über</string>
+
+    <!--titles-->
+    <string name="fragment_donate_title">Spenden</string>
+    <string name="fragment_donate_clipboard_label">Adresse kopiert</string>
+    <string name="fragment_donate_clipboard_message">Adresse in die Zwischenablage kopiert</string>
+    <string name="fragment_about_title">Über</string>
+
+    <!--dialog fragments-->
+    <string name="dialog_favorites_title">Favorisierte Orte</string>
+    <string name="dialog_favorites_empty_view">Leer</string>
+    <string name="dialog_favorites_add_hint">Name</string>
+    <string name="dialog_favorites_add_lat_long">Breitengrad: %1$f \nLängengrad: %2$f</string>
+    <string name="dialog_favorites_list_name_hint">Name</string>
+    <string name="dialog_favorites_list_edit_desc">Bearbeiten</string>
+    <string name="dialog_favorites_list_del_desc">Löschen</string>
+
+    <string name="dialog_go_to_title">Koordinaten</string>
+    <string name="dialog_go_to_lat_hint">Breitengrad</string>
+    <string name="dialog_go_to_long_hint">Längengrad</string>
+    <string name="dialog_go_to_invalid_range">Ungültiger Bereich</string>
+    <string name="dialog_go_to_invalid_number">Ungültige Nummer</string>
+    <string name="dialog_go_to_invalid_empty">Darf nicht leer sein</string>
+
+    <!--main activity dialogs-->
+    <string name="dialog_first_run_title">Standort Einstellungen</string>
+    <string name="dialog_first_run_message">Wenn Sie Probleme damit haben, dass Ihr tatsächlicher Standort offengelegt wird, insbesondere bei Google Maps-basierten Apps und Diensten,
+        wird empfohlen, alle Einstellungen in den Standort-Einstellungen zu deaktivieren, die Wi-Fi-Punkte und mehr verwenden, um Ihren Standort schneller und genauer zu bestimmen.</string>
```

Comment:
Change "Wi-Fi-Punkte" to the anglicism "Wireless Access Points" or the German word "WLAN-Zugangspunkte" respectively "drahtlose Zugangspunkte".